### PR TITLE
[feature support]add CIT

### PR DESCRIPTION
**Pull request by waitfor-night**

### CODE REVIEW

**Review by merlintang — CHANGES_REQUESTED** [review]
can you provide more details about the CIT meaning, and update the readme?

File: mlora/config/task.py [hunk 1]
```diff
@@ -121,8 +121,32 @@ def __init__(
...
+        self,
+        config: Dict[str, str],
+        adapters: Mapping[str, AdapterConfig],
+        datasets: Mapping[str, DatasetConfig],
+    ):
+        super().__init__(config, adapters, datasets)
+        self.init(self.__params_map, config)
+
+        self.lambda_ = float(self.lambda_)
+
```
Comment (yezhengmao1) [inline mlora/config/task.py:145]: temperature show cast to float

File: mlora/executor/task/cit_task.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+from .train_task import TrainTask
+
+
+class CITTask(TrainTask):
+    context_: TrainLoRAContext
+    config_: CITTaskConfig
+    pooling_method_: str
+
+    @override
```
Comment (yezhengmao1) [inline mlora/executor/task/cit_task.py:21]: do not override this function, seem same

**Review by yezhengmao1 — CHANGES_REQUESTED** [review]

**Comment by yezhengmao1** [comment]
wait the ci: https://github.com/TUDB-Labs/mLoRA/actions/runs/9825498507

**Review by merlintang — APPROVED** [review]

**Pull request merged**